### PR TITLE
Izi Refactoring(Administration menu)

### PR DESCRIPTION
**Pull request by muarachmann**
@aethelwulffe here is a cleaner branch to clarify the #1136  will push them gradually

### CODE REVIEW

**Comment by muarachmann** [comment]
screenshot from 2018-07-18 06-59-54

is this popup necessary if yes please let me know i dont see what it is used for in the codebase

File: interface/main/tabs/edit_roles.php [hunk 1]
```diff
@@ -59,35 +59,82 @@
...
-<?php call_required_libraries(array("jquery-min-3-1-1","bootstrap","fancybox-custom"));
+<?php call_required_libraries(array("jquery-min-3-1-1","bootstrap","font-awesome","iziModalToast"));
       resolveFancyboxCompatibility();
 ?>
     <title><?php echo xlt("Role Management") ?></title>
     <link href="js/jsoneditor/jsoneditor.css" rel="stylesheet" type="text/css">
     <script src="js/jsoneditor/jsoneditor.js"></script>
     <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/common.js"></script>
     <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-ui.js"></script>
-    <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery.easydrag.handler.beta2.js"></script>
```
Comment (aethelwulffe) [inline interface/main/tabs/edit_roles.php]: Did you intend to remove easydrag?  This is just applicable to the fancybox, right?

File: interface/main/tabs/edit_roles.php [hunk 2]
```diff
@@ -118,15 +168,15 @@
             <?php foreach($role_list as $role_title) { ?>
             <tr>
                 <td> <span class="text">  <?php echo $role_title; ?> </span> </td>
-                <td> <a href="../../roles/role_edit.php?title=<?php echo $role_title; ?>"  class="iframe_medium" onclick="top.restoreSession()"> Edit  </a> </td>
```
Comment (aethelwulffe) [inline interface/main/tabs/edit_roles.php]: This part of the PR?

Comment (aethelwulffe) [reply interface/main/tabs/edit_roles.php]: Guess it is.

**Comment by muarachmann** [comment]
@aethelwulffe, @teryhill , @tmccormi check this out

**Comment by muarachmann** [comment]
screenshot from 2018-08-13 22-52-47

Message alert on sending messages. Noticed that messages are being sent even though the text area field is empty. So prevented that.Also,I showed alert messages in case a "sender to" is not selected or patient . Success message returned upon a successful form submission.

**Comment by muarachmann** [comment]
screenshot from 2018-08-13 23-31-16

Worked on Adding issues , editing and opening encounters from the patient demo graphic screen. Will fo so for the menu too

**Comment by muarachmann** [comment]
Worked also on edit users/add users 
screenshot from 2018-08-13 23-32-43

**Comment by aethelwulffe** [comment]
@muarachmann Could you please look at the conflict here?  Seems simple...but I don't have time to delve into the blame to check where the conflict happened.  May need some change.

**Comment by muarachmann** [comment]
Okay i deleted my repo if you notice this shows unkown repo. I have to get to my linux machine to figure that out.

**Comment by muarachmann** [comment]
If you could show me the screenshot i could fix it here and  u merge, I dont know a means of getting my old branch back except you have a way of me pulling this PR into a branch

**Comment by aethelwulffe** [comment]
OK.  
@muarachmann please reference #1395

**Comment by aethelwulffe** [comment]
This had a replacement PR.  Should have been closed.

**Pull request closed without merging**